pinot-core/src/main/java/org/apache/pinot/core/operator/DocIdSetOperator.java

```diff
@@ -34,6 +37,7 @@
  */
 public class DocIdSetOperator extends BaseOperator<DocIdSetBlock> {
   private static final String OPERATOR_NAME = "DocIdSetOperator";
+  private static final String EXPLAIN_NAME = null;
```

Comment:
Why null ? Is it because we want to skip this operator ?

Reply:
Yes, for an operator to be included in EXPLAIN PLAN output, it must define an EXPLAIN PLAN name.

Reply:
There isn't any big reason behind it, except that we didn't think this would be useful enough to show in the EXPLAIN PLAN output. @siddharthteotia ?

In either case, I would suggest that we not get into particular operators of EXPLAIN PLAN and output display format etc. in this PR. As the PR states EXPLAIN PLAN output is only for information purposes and we are not guaranteeing that the output will remain backward compatible. So we can change what is displayed in the EXPLAIN PLAN output anytime as needed.

Reply:
Sure, I don't see this as a blocker, but I do feel this level is useful to understand the query execution layers, so suggest including it in the explain tree

Reply:
I agree this should be part of the tree.